files/restore_backup.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
Uhm, in the guidelines it says I should use #!/usr/bin/env
https://github.com/ansible/ansible/blob/devel/CODING_GUIDELINES.md